sup3r/utilities/plot_utilities.py

```diff
@@ -0,0 +1,384 @@
+# -*- coding: utf-8 -*-
+"""Utilities module for plotting data
+
+@author: malihass
+"""
+
+import imageio
+import matplotlib
+from matplotlib import cm
+import matplotlib.pyplot as plt
+import numpy as np
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+
+def prettyLabels(xlabel, ylabel, fontsize=14, title=None):
```

Comment:
should use underscore for function names with multiple words, as per python naming conventions

Reply:
snake_case